daprdocs/content/en/operations/observability/tracing/otel-collector/open-telemetry-collector-jaeger.md

```diff
@@ -58,41 +61,103 @@ To view traces in your browser, go to `http://localhost:16686` to see the Jaeger
 
 {{% tab "Kubernetes" %}}
 <!-- kubernetes -->
-## Configure Jaeger on Kubernetes with the OpenTelemetry Collector
+## Configure Jaeger V2 on Kubernetes
 
-The following steps show you how to configure Dapr to send distributed tracing data to the OpenTelemetry Collector which, in turn, sends the traces to Jaeger.
+The following steps show you how to configure Dapr to send distributed tracing data directly to a Jaeger V2 instance deployed using the OpenTelemetry Operator with in-memory storage.
 
 ### Prerequisites
 
 - [Install Dapr on Kubernetes]({{% ref kubernetes %}})
-- [Set up Jaeger](https://www.jaegertracing.io/docs/1.49/operator/) using the Jaeger Kubernetes Operator
-
-### Set up OpenTelemetry Collector to push to Jaeger
 
-To push traces to your Jaeger instance, install the OpenTelemetry Collector on your Kubernetes cluster.
+### Set up Jaeger V2 with the OpenTelemetry Operator
 
-1. Download and inspect the [`open-telemetry-collector-jaeger.yaml`](/docs/open-telemetry-collector/open-telemetry-collector-jaeger.yaml) file.
+Jaeger V2 can be deployed using the OpenTelemetry Operator for simplified management and native OTLP support. The following example configures Jaeger V2 with in-memory storage.
 
-1. In the data section of the `otel-collector-conf` ConfigMap, update the `otlp/jaeger.endpoint` value to reflect the endpoint of your Jaeger collector Kubernetes service object.
+> **Note on Storage Backends:** This example uses in-memory storage (`memstore`) for simplicity, suitable for development or testing environments as it stores up to 100,000 traces in memory. For production environments, consider configuring a persistent storage backend like Cassandra or Elasticsearch to ensure trace data durability.
 
-1. Deploy the OpenTelemetry Collector into the same namespace where your Dapr-enabled applications are running:
+#### Installation
 
-   ```sh
-   kubectl apply -f open-telemetry-collector-jaeger.yaml
+1. **Install cert-manager** to manage certificates:
+   ```bash
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.16.1/cert-manager.yaml -n cert-manager
```

Comment:
I think 1.16.5 is the latest, no? Where did you get that version? Yesterday I was looking at the releases page and I found `1.16.5` I guess that they are not different.